test/e2e/suites/managed/helpers.go

```diff
@@ -121,7 +121,7 @@ func verifyConfigMapExists(ctx context.Context, name, namespace string, k8sclien
 	cm := &corev1.ConfigMap{}
 	Eventually(func() error {
 		return k8sclient.Get(ctx, apimachinerytypes.NamespacedName{Name: name, Namespace: namespace}, cm)
-	}, 2*time.Minute, 5*time.Second).Should(Succeed())
+	}, 5*time.Minute, 5*time.Second).Should(Succeed())
```

Comment:
I would appreciate a comment explaining why the timeout is what it is (looks like a shorter timeout causes e2e to sometimes fail?)

Reply:
you know, I was trying to figure out why the configmap `aws-auth` was missing but it seems like it was because the EKS clusters did not have nodes attached to them and so the configmap was not created.

now since i am adding a default it seems to make everything better. so i will remove this increased timeout